### PR TITLE
[MRRESOURCES-154] Allow to require project properties by bundle

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: src/main/java/org/apache/maven/plugin/resources/remote/BundleRemoteResourcesMojo.java [hunk 1]
```diff
@@ -87,6 +86,15 @@ public class BundleRemoteResourcesMojo extends AbstractMojo {
     @Parameter(defaultValue = "${project.build.sourceEncoding}")
     private String sourceEncoding;
 
+    /**
+     * List of required project properties needed to process Velocity
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugin/resources/remote/BundleRemoteResourcesMojo.java:90]: delete "required" as it's redundant with needed

File: src/main/mdo/remote-resources.mdo [hunk 2]
```diff
@@ -52,12 +52,23 @@ under the License.
...
+          <version>1.1.0+</version>
           <description><![CDATA[
             Source encoding of the remote resources contained in the bundle.
           ]]></description>
           <type>String</type>
         </field>
+        <field>
+          <name>requiredProjectProperties</name>
+          <version>1.2.0+</version>
+          <description><![CDATA[
```
Comment (elharo) [inline src/main/mdo/remote-resources.mdo:64]: CDATA not needed here

Comment (slawekjaranowski) [reply src/main/mdo/remote-resources.mdo]: removed and in other places also

**Review by slachiewicz — APPROVED** [review]

**Review by elharo — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #200

**Pull request merged**